### DISCUSSION TITLE
level 5 badge + extra cards in inventory

### DISCUSSION THREAD

**Discussion by catharsis71** [Rudokhvist/ASF-STM#38] [General]
I ended up with a few games where I finished the level 5 badge but still have enough extra cards to make an additional set

I would like to finish these sets (since there are tradebots that offer set4set trades) however, ASF-STM appear to ignore these "extra" potential sets

researching further, I found it is possible to determine whether extra cards exist just by checking the badge listing page.   if a level 5 badge says "no card drops remaining", that means there are extra cards in the inventory.  for example:

![image](https://github.com/user-attachments/assets/e71e8205-3feb-4411-b212-937322851d91)

this means that "Down to One" still has cards in inventory, and the other two games do not

extra cards on a completed level 5 badge look like this:

![image](https://github.com/user-attachments/assets/8a7d33f5-a79f-45bd-a76d-6d66b4ba406d)

I know this is kind of a niche situation, but it would be nice if ASF-STM could recognize that "level 5 badge + 'no card drops remaining' = extra cards in inventory" and put that game in the queue for scanning & potential matchmaking

**Comment by Rudokhvist** [comment]
This is already in TODO list, but since I'm pretty busy recently - I can't give no estimate of when (and even if) it will be done.

**Comment by YrShorts** [comment]
I am looking for this feature. Can something like this JSON be added to ASF STM? It may fumble foil sets without additional logic.

{
  "Badges": {
    "Level": 5,
    "IncludeFoil": false
  },
  "Inventory": {
    "HasCards": true
  }
}

**Reply by Rudokhvist** [reply]
Honestly, I have no idea what you are talking about. Could you please elaborate?
